Cargo.toml

```diff
@@ -85,12 +85,11 @@ version = "0.7.0"
 license = "Apache-2.0"
 
 [workspace.dependencies]
-rocksdb = { version = "0.22", features = [
-  # "multi-threaded-cf",
-] }
+rocksdb = { version = "0.22", git = "https://github.com/madara-alliance/rust-rocksdb", branch = "read-options-set-raw-snapshot" }
+librocksdb-sys = { version = "0.17.0", git = "https://github.com/madara-alliance/rust-rocksdb", branch = "read-options-set-raw-snapshot" }
```

Comment:
im not sure why they do that but they do that upstream, their bindings and rocksdb crate have different versions
might be because librocksdb-sys follows the official rocksdb versions and the rocksdb crate doesnt